fetch.bs

```diff
@@ -3984,6 +3987,10 @@ the request.
   <ol>
    <li><p>Let <var>value</var> be `<code>*/*</code>`.
 
+
+   <li><p>If <var>request</var>'s <a for=request>initiator</a> is "<code>prefetch</code>", then set
+   <var>value</var> to <a lt="document Accept header value">document `Accept` header value</a>.
+
    <li>
     <p>A user agent should set <var>value</var> to the first matching statement, if any, switching
```

Comment:
This needs "Otherwise" in front or it will get overwritten, no?

Reply:
Done